src/content/migrate/3.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: To webpack v2, v3
-sort: 1
+title: To v2, v3
```

Comment:
I don't understand previous title, neither this one. It is too cryptic :D

Reply:
when you open the page it makes sense: https://webpack.js.org/migrate/